src/components/Community/data.json

```diff
@@ -1,7 +1,7 @@
 {
   "hero": {
-    "pictureDesktop": "/img/community/banner.png",
-    "pictureMobile": "/img/community/banner-mobile.png",
+    "pictureDesktop": "/img/community/header_mailing_list.png",
```

Comment:
can we get the same with a plain white background? (and may be w/o the DVC logo)

Reply:
just pushed a white banner